packages/@coorpacademy-components/src/organism/review-stacked-slides/style.css

```diff
@@ -42,35 +40,35 @@
   z-index: 50;
   width: 100%;
   transform: translate(0, 0);
-  transition: transform 0.5s ease-in;
+  transition: transform 1s ease-in, width 1s ease-in;
```

Comment:
This changes should not been here in your PR, please check when you build your PR which files have been added. You must be sure that the PR does not includes files changed by other PRs.